_services/hubspot.html

```diff
@@ -76,78 +88,91 @@ <h2 class="content-row__title">Our Services</h2>
+          <p class="quote-text">
+            228,000+ customers in over 135 countries grow their businesses with HubSpot
+          </p>
+          <p class="quotee"><a href="https://www.hubspot.com/why-choose-hubspot">Source: HubSpot</a></p>
+      </section>
+      <p>
+        The software is <strong>easy to use</strong>, <strong>fast</strong> and <strong>unified</strong> - the tools are powerful alone, but better together. So if you're looking for a single source of truth that integrates with over 1500+ apps - we recommend getting in touch.
+        </p>
     </div>
   </section>
 </div>
 
 <div class="content-row">
   <section class="content-row__section">
     <img src="/img/extension-icon.svg" class="content-row__icon" alt="" />
-    <h2 class="content-row__title">HubSpot Features</h2>
+    <h2 class="content-row__title">Setup</h2>
     <div class="content-row__text">
       <p>
-        HubSpot is packed full of features, to help you optimize your business.
-        From lead management & prospecting tools to Sales Automation, Email
-        Tracking and HubSpot’s AI that helps you craft compelling sales emails for
-        your outreach.
+        HubSpot is feature packed, but you've got to start somewhere - we can help you decide what to implement first.
       </p>
-      <p>Here are six of HubSpot’s key features:</p>
       <p>
-        <strong>Sales Hub</strong> - Simple to use CRM that leverages AI to help
-        you build efficient sales pipelines and close more deals.
-        <a href="https://www.hubspot.com/products/sales" target="_blank"
-          >Read More</a
-        >
+        We will:
       </p>
       <p>
-        <strong>Marketing Hub</strong> - Marketing software powered by AI helping
-        you generate leads and automate your marketing campaigns.
-        <a href="https://www.hubspot.com/products/marketing" target="_blank"
-          >Read More</a
-        >
+        <ul>
+          <li>Define the specific needs of your business with you</li>
+          <li>Define what brings you value - fast</li>
+          <li>Onboard and train you how to get the most out of HubSpot</li>
+        </ul>
       </p>
+      <p>Our Top Tip is to <strong>Start small and scale up</strong>. In our experience Sales Hub coupled with Breeze AI helps businesses streamline and grow our client's sales pipelines, generating revenue for setting up other hubs.</p>
+      </p>
+        <h3>Key HubSpot features</h3>
       <p>
-        <strong>Service Hub</strong> - Harness the power of AI to help scale your
-        support operations.
-        <a href="https://www.hubspot.com/products/service" target="_blank"
-          >Read More</a
-        >
+        <strong>Breeze AI</strong> - AI made easy -
+        an AI solution that delivers ease, speed, and unification across your business.
+        <a href="https://www.hubspot.com/products/artificial-intelligence" target="_blank"
+          >Read More</a>
```

Comment:
We can discuss if it make sense from SEO perspective to replace 'Read more' links text here

Reply:
Changed to link in titles.